04 Strategy Library/999 Seasonality and Session Effect in Index Futures/04 Conclusion.html

```diff
@@ -1,7 +1,7 @@
 <p>
-For Night Session Strategy, both in the Bullish Season and the Bearish Season, the Night Session effect (well trading performance if buy at its open and sell at its end) for E-mini S&P 500 futures exists based on an 8-year long backtest. Particularly, on the bearish Season, we gain a 13%  return only with 5% maximum drawdown. 
+	As can be seen in the backtest results, trading just during the Night Session, in both market seasons, performs well over an extended period. This strong performance is particularly true during a Bearish Season, where we can achieve 13% returns with a small drawdown of only 5%.
 
-For the Combined Strategy, its performance is not good, which is not consistent with the two articles on Seasonality and Market Session Effect. The reason is in the Afternoon Session and Close Session, the return is not as high as expected, and hence the Combined Strategy underperforms compared to the Night Session-only strategy.
+	Despite our initial intuition, the combined strategy performs poorly after running a backtest. Despite trading during the top-performing sessions of both seasons, we don't see nearly the level of performance we would have expected, and it underperforms trading just the Night Session. The likely reason is that the Afternoon Session and Close Session underperform their theoretical results and thus drag down the combined performance.
```

Comment:
"Despite our initial intuition," --> "Contrary to our initial intuition,"

Sorry, I should have caught this in my own comment. "Despite" is used twice in a row to begin sentences, not good writing.